python-threatexchange/threatexchange/exchanges/impl/ncmec_api.py

```diff
@@ -39,14 +39,16 @@ class NCMECCheckpoint(
 
     # The biggest value of "to", and the next "from"
     get_entries_max_ts: int
+    next_fetch: str
+    last_fetch_time: int
```

Comment:
this is unsued currently but I know we talked about invalidating checkpoints older than 24hr.. happy to add that! just wanted to get confirmation on the checkpointing piece that this is on the right track